Source/WebKit/WebProcess/WebCoreSupport/WebLocalFrameLoaderClient.cpp

```diff
@@ -2030,10 +2032,19 @@ void WebLocalFrameLoaderClient::didAccessWindowProxyPropertyViaOpener(WebCore::S
 
 void WebLocalFrameLoaderClient::frameNameChanged(const String& frameName)
 {
+    if (siteIsolationEnabled())
+        return;
```

Comment:
I think this should be `if (!siteIsolationEnabled())`

Reply:
fixed - thanks.